pkg/api/errors/errors.go

```diff
@@ -25,6 +25,11 @@ import (
 	"github.com/GoogleCloudPlatform/kubernetes/pkg/util/errors"
 )
 
+// HTTP Status code not in the golang http package.
+const (
+	TooManyRequests = 429
```

Comment:
Please prefix with "Status".

Please also move StatusUnprocessableEntity to here:
https://github.com/GoogleCloudPlatform/kubernetes/blob/master/pkg/apiserver/apiserver.go#L54